apps/api/src/app/events/usecases/send-message/send-message-in-app.usecase.ts

```diff
@@ -32,6 +32,7 @@ import { CacheKeyPrefixEnum, InvalidateCacheService } from '../../../shared/serv
 import { SendMessageBase } from './send-message.base';
 import { ApiException } from '../../../shared/exceptions/api.exception';
 import { OrganizationEntity } from '../../../../../../../libs/dal/src/repositories/organization/organization.entity';
```

Comment:
I know not related but this import is bad, need to be refactored to be from @novu/dal, don't we have a lint rule for banned imports 🤔

Reply:
I'll fix this one, but agree we should enable that rule.